clj-refactor.el

```diff
@@ -1952,20 +1944,15 @@ root."
   (let* ((symbol (cider-symbol-at-point))
          (var-info (cider-var-info symbol))
          (symbol-name (nrepl-dict-get var-info "name"))
-         (ns (nrepl-dict-get var-info "ns")))
-    (let* ((occurrences (-> (or symbol-name symbol)
-                            (cljr--find-symbol-sync ns)
-                            cljr--read-symbol-metadata))
-           (buffer-of-symbol (cider-find-var-file (concat ns "/" symbol-name)))
-           (tooling-buffer-p (cider--tooling-file-p (buffer-name buffer-of-symbol))))
-      (cljr--rename-symbol ns occurrences new-name)
-      (when (and (not tooling-buffer-p) symbol-name)
-        (message "Reloading buffer %s where symbol is defined" buffer-of-symbol))
-      (message "Renamed %s occurrences of %s" (length occurrences) (or symbol-name symbol))
-      (when (not tooling-buffer-p)
-        (when symbol-name
-          (cider-load-buffer buffer-of-symbol))
```

Comment:
I removed the loading of buffers @benedekfazekas.  I don't see the point in reloading only the buffer from which the request was sent.  If anything should be done at all every buffer that's been touched should be reloaded.  We might take it upon ourselves to call `cider-refresh` here.  What do you think?